src/components/Navigation/index.js

```diff
@@ -0,0 +1,47 @@
+import { AppBar, Hidden, Toolbar } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import React from "react";
+
+import DesktopNavigation from "./DesktopNavigation";
+import MobileNavigation from "./MobileNavigation";
+
+const useStyles = makeStyles(({ typography, breakpoints, palette }) => ({
+  root: {
+    backgroundColor: palette.background.default,
+  },
+  section: {
+    padding: `0 ${typography.pxToRem(21)} 0 ${typography.pxToRem(17)}`,
+    margin: 0,
+    width: "100%",
+    [breakpoints.up("lg")]: {
+      padding: `0 0 0 ${typography.pxToRem(100)}`,
+      margin: "0 auto",
+      maxWidth: typography.pxToRem(1196),
+    },
+  },
```

Comment:
@nerdlet we should not override the styles for section here. Nav takes full section width (1160 for lg)
![Screenshot from 2021-07-28 07-09-33](https://user-images.githubusercontent.com/7962097/127262681-45c03dcc-1d72-446c-800c-1c331b000661.png)